src/alert-manager/src/alert-handler/controllers/node.js

```diff
@@ -72,7 +73,110 @@ const cordonNodes = (req, res) => {
     });
 };
 
+const getK8sV1Job = (jobName, nodeName, minorNumber) => {
+  const DOCKER_REGISTRY_PREFIX = process.env.DOCKER_REGISTRY_PREFIX;
+  const DOCKER_REGISTRY_TAG = process.env.DOCKER_REGISTRY_TAG;
+  const job = {
+    apiVersion: 'batch/v1',
+    kind: 'Job',
+    metadata: {
+      name: jobName,
+    },
+    spec: {
+      // TTL feature is currently alpha[Kubernetes 1.15]
+      // To avoid using this fearure, jobs with label `time-to-live=24h` ill be cleaned with function `cleanTTL24HJobs` regularlly
+      // ttlSecondsAfterFinished: 86400,
+      template: {
+        metadata: {
+          name: 'nvidia-gpu-low-perf-fixer',
+          labels: {
+            'time-to-live': '24h',
+          },
```

Comment:
Do we need metadata here? This seems should be generated automatically

Reply:
> Do we need metadata here? This seems should be generated automatically

Removed, thx